.github/workflows/rolling-release.yml

```diff
@@ -0,0 +1,96 @@
+name: Rolling Release Workflow
+
+on:
+  push:
+    branches: [master, 6.0/stage, 6.0/release]
+
+jobs:
+  deb-build-test-release:
+    name: Debian Package Build->Test->Release
+    #
+    # In an attempt to ensure no surprises down the line we
+    # attempt to run everything in the latest version of
+    # Ubuntu that Github provides.
+    #
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Setup Python
+        uses: actions/setup-python@v1
+        with:
+          python-version: '3.6'
+      - name: Detect Branch & Set Environmet Variables
+        run: |
+          echo "Working with: ${{ github.ref }}"
+          #
+          # Normalized names are mapped like the following examples:
+          #   "refs/head/master" -> "master"
+          #   "refs/head/6.0/stage" -> "6-0_release"
```

Comment:
nit: `6-0_release` -> `6-0_stage`. Also, have you though of simply deleting those characters, i.e. `6.0/stage` -> `60stage`

Reply:
Good catch!

I thought about deleting them but decided not to and the point was the ambiguity of triple digits like 333 - does it mean 3.33 or 33.3? I know this probably isn't going to be a problem given the numbers that we are right now but I don't think keeping that separation would hurt.